script.js

```diff
@@ -1,4 +1,145 @@
+
+  var newFormat = format();
+  console.log("format: " + newFormat);
+
+  //program compares generated rsp to choice
+  //draw condition
+  if (
+    (userInput == "scissors" && gameResult == "scissors") ||
+    (userInput == "paper" && gameResult == "paper") ||
+    (userInput == "stone" && gameResult == "stone")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> It's a draw. <br> Play again?`;
+  }
+  // win condition
+  if (
+    (userInput == "scissors" && gameResult == "paper") ||
+    (userInput == "paper" && gameResult == "stone") ||
+    (userInput == "stone" && gameResult == "scissors")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> You win. Play again?`;
+  }
+
+  // lose condition
+  if (
+    (userInput == "scissors" && gameResult == "stone") ||
+    (userInput == "paper" && gameResult == "scissors") ||
+    (userInput == "stone" && gameResult == "paper")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}!<br> You lose. Play again?`;
+  }
+  //**REVERSE GAME */
+  //draw condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "scissors") ||
+    (userInput == "reverse paper" && gameResult == "paper") ||
+    (userInput == "reverse stone" && gameResult == "stone")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> It's a draw. <br> Play again?`;
+  }
+  // lose condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "paper") ||
+    (userInput == "reverse paper" && gameResult == "stone") ||
+    (userInput == "reverse stone" && gameResult == "scissors")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}! <br> You lose. Play again?`;
+  }
+
+  // win condition
+  if (
+    (userInput == "reverse scissors" && gameResult == "stone") ||
+    (userInput == "reverse paper" && gameResult == "scissors") ||
+    (userInput == "reverse stone" && gameResult == "paper")
+  ) {
+    return `You chose ${newFormat}. <br> The AI chose ${rspFormat}!<br> You lose. Play again?`;
+  }
+  //**REVERSE GAME */
+
+  // if wrong input, prompt user to try again
+  else if (
+    userInput != "scissors" ||
+    userInput != "paper" ||
+    userInput != "stone" ||
+    userInput != "reverse scissors" ||
+    userInput != "reverse paper" ||
+    userInput != "reverse stone"
+  ) {
+    return `Oops! You have input the wrong option. Please only type in "scissors" or "paper" or "stone" to join the game. Try again!`;
+  }
+  return rspResult;
+};
+
+// program generates rsp
+
+var aiResult = function () {
+  var rspNumber = 1 + Math.floor(Math.random() * 3);
+  console.log(rspNumber);
+
+  if (rspNumber == 1) {
+    rspResult = "scissors";
+    rspFormat = "scissors ✂️";
+  }
+
+  if (rspNumber == 2) {
+    rspResult = "paper";
+    rspFormat = "paper 🗒";
+  }
+
+  if (rspNumber == 3) {
+    rspResult = "stone";
+    rspFormat = "stone O";
+  }
+  return rspResult;
+};
```

Comment:
Great job decomposing the computer's possibles moves all in 1 function and assigning this to a variable 💪